Sources/PluginLibrary/NamingUtils.swift

```diff
@@ -70,6 +69,32 @@ fileprivate let reservedTypeNames: Set<String> = {
   return names
 }()
 
+/*
+ * Many Swift reserved words can be used as fields names if we put
+ * backticks around them:
+ */
+fileprivate let quotableFieldNames: Set<String> = {
+  () -> Set<String> in
+  var names: Set<String> = []
+
+  // We don't need to protect all of these keywords, just the ones
+  // that interfere with enum cases:
+  // names = names.union(swiftKeywordsReservedInParticularContexts)
```

Comment:
Done.  Turns out I never cycled back to finish validation, these ones didn't need to be here.